src/sentry/static/sentry/app/components/events/contexts/redux.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {t} from 'app/locale';
+import ContextBlock from 'app/components/events/contexts/contextBlock';
+import {KeyValueListData} from 'app/components/events/interfaces/keyValueList/types';
+
+type Props = {
+  alias: string;
+  data: Record<string, any>;
+};
+
+class ReduxContextType extends React.Component<Props> {
+  getKnownData(): KeyValueListData[] {
+    return [
+      {
+        key: 'value',
+        subject: t('Value'),
```

Comment:
```suggestion
        subject: t('Latest State'),
```
Just minor but can we have it say Latest State instead of Value? Looks great!